ui/appui-react/src/appui-react/theme/ThemeManager.tsx

```diff
@@ -13,18 +13,51 @@ import { ThemeProvider } from "@itwin/itwinui-react";
 import type { FrameworkState } from "../redux/FrameworkState";
 import { UiFramework } from "../UiFramework";
 
+/** System preferred color theme.
+ * @public
+ */
+export const SYSTEM_PREFERRED_COLOR_THEME = "SYSTEM_PREFERRED";
+
 /** Enum for the Color Theme string.
  * @public
  */
 export enum ColorTheme {
   Light = "light",
   Dark = "dark",
+  System = SYSTEM_PREFERRED_COLOR_THEME,
+  Inherit = "inherit",
+  HighContrastLight = "high-contrast-light",
+  HighContrastDark = "high-contrast-dark",
 }
 
-/** System preferred color theme.
+/**
+ * List of valid theme values
```

Comment:
```suggestion
 * Describes valid theme types.
```

Reply:
Changed (but did not put in "types", as "A type that describe a type" seems too much.)